app/views/warehouse_reports/project/data_qualities/_organization.haml

```diff
@@ -35,7 +35,7 @@
               = check_box_tag("project[#{project.id}]", 1, false, {class: 'j-child-select'})
               %label{for:"project_#{project.id}", tabindex:'1' }
           %td<
-            = link_to project.ProjectName, project_data_quality_reports_path(project)
+            = link_to project.name, project_data_quality_reports_path(project)
```

Comment:
Should we pass in `current_user`?

Reply:
This report will only be visible for confidential projects _if_ the user has permission to see confidential projects. So `project.name` and `project.ProjectName` are effectively the same. I can update it to `name(current_user)` in case we change that down the line?